airbyte-integrations/connectors/source-sendgrid/unit_tests/unit_test.py

```diff
@@ -30,3 +41,14 @@ def test_source_wrong_credentials():
     source = SourceSendgrid()
     status, error = source.check_connection(logger=AirbyteLogger(), config={"apikey": "wrong.api.key123"})
     assert not status
+
+
+def test_messages_stream(mock_pendulum_now):
+    start_time = 1558359837
+    stream = Messages(start_time)
+    state = {"last_event_time": 1558359000}
+    request_params = stream.request_params(state)
+    assert (
+        request_params
```

Comment:
@alafanechere do you know why this is returning a string instead of a Mapping[str, Any]?
If there's no obvious reason, I'd like to change it to a mapping so it's consistent with the interface

Reply:
No I don't see obvious reason, the URL formatting could be a bit tricky due to the API query format, but I don't see any good reason to not use a mapping.